sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/TrampolineUtil.scala

```diff
@@ -75,4 +76,10 @@ object TrampolineUtil {
   def cleanupAnyExistingSession(): Unit = SparkSession.cleanupAnyExistingSession()
 
   def asNullable(dt: DataType): DataType = dt.asNullable
+
+  /** Return true if the Attribute passed is one of aggregates in the list */
+  def validateAggregate(agg: Attribute): Boolean = {
+    val allAggs = List("min", "max", "avg", "sum", "count", "first", "last")
+    allAggs.contains(agg.toString().split("\\(")(0))
```

Comment:
Why do we need the split, can we do agg.contains(any of these list items) ?

Reply:
The attribute returned would include columnName and type - `first(testtable.`ints`)#164` , so splitting it on "(" as delimiter to get the aggregate name.

Reply:
We _need_ to find a simpler way to do this using scala forall or something similar so that we dont have to do any string magic.
U can turn it on its head and use `agg` to do string matching